### PR TITLE
Ci 378 fix guide landing page spacing

### PR DESCRIPTION
**Pull request by jessgilbert**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR

image

### CODE REVIEW

File: core/sass/partials/_utils.scss [hunk 1]
```diff
@@ -139,6 +139,15 @@
   }
 }
 
+
+@media (max-width: 961px) {
+  .reverse-columns-xl {
```
Comment (Kenneth-Siu) [inline core/sass/partials/_utils.scss]: I think this is actually `.reverse-columns-l`.

Following from the other one's pattern, this is everything l and smaller.

File: core/sass/partials/_utils.scss [hunk 2]
```diff
@@ -139,6 +139,15 @@
   }
 }
 
+
+@media (max-width: 961px) {
```
Comment (Kenneth-Siu) [inline core/sass/partials/_utils.scss]: max-width: 960px

It's min-width 961, but max-width 960

File: core/sass/partials/_utils.scss (hunk 1 above)
Comment (jessgilbert) [reply core/sass/partials/_utils.scss]: Would it not be slightly confusing that it uses `column-half-xl` but `reverse-column-l` which both have the same break points?

Comment (jessgilbert) [reply core/sass/partials/_utils.scss]: Isn't the large break point ` (min-width: 769px)`?

**Comment by jessgilbert** [comment]
image
image

**Pull request merged**